starlite/kwargs.py

```diff
@@ -138,6 +142,8 @@ def create_for_signature_model(
             else:
                 parameter_set = expected_query_parameters
                 field_alias = model_info.extra.get("query") or field_name
+                if model_field.shape == SHAPE_LIST:
```

Comment:
What about:

```python
    SHAPE_SEQUENCE,
    SHAPE_SET,
    SHAPE_TUPLE
```

How will this behave with them? And what is the expected behaviour? I would submit that this should be more generic than a list - it should work on all sequence types. And the tests should cover this.